R/course.R

```diff
@@ -154,6 +155,27 @@ use_zip <- function(url,
 #' https://api.github.com/repos/r-lib/usethis/zipball/master
 #' ```
 #'
+#' **Google Drive:**
+#'
+#' It looks like it's not possible to directly share as a ZIP file the content
+#' of a folder storred in Google Drive. It is however possible to share a ZIP
```

Comment:
Bummer. I really thought this was possible, but from my own searching, I agree there doesn't seem to be a workflow that is comparable to what we have for DropBox or GitHub.